vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -1112,18 +1118,18 @@ private void createAnalysisRecords(
                                                 })
                                         .collect(Collectors.toList())))))));
 
-      context
-          .select(DSL.jsonObject(fields))
-          .from(WORKFLOW_RUN)
-          .where(condition)
-          .forEach(
-              result -> {
-                try {
-                  jsonGenerator.writeRawValue(result.value1().data());
-                } catch (IOException e) {
-                  throw new RuntimeException(e);
-                }
-              });
+    context
+        .select(DSL.jsonObject(fields))
```

Comment:
i thought this pr is to remove this?

Reply:
Weirdly, no. Only certain calls to `DSL.jsonObject` are deprecated. The array form is. The one that takes `JSONEntry` is fine.
https://www.jooq.org/javadoc/latest/org.jooq/org/jooq/impl/DSL.html#jsonbObject(org.jooq.Field...)

Reply:
Ohh I see. the formatting made github not highlight the other change and i missed it. sounds good!

Reply:
Ah, sorry.